packages/server/src/server/handlers/harness.ts

```
@@ -0,0 +1,1199 @@
+import type { Harness, Session } from '@mastra/core/harness';
+import type { Agent } from '@mastra/core/agent';
```

ESLint

[error] `@mastra/core/agent` type import should occur before type import of `@mastra/core/harness`

(import/order)